pkg/installer/install.go

```diff
@@ -250,12 +249,37 @@ func (in *Installer) installStorageOS() error {
 	if err = in.operatorServicesAreReady(filepath.Join(stosDir, operatorDir, stosOperatorFile)); err != nil {
 		return err
 	}
+	if in.stosConfig.Spec.Install.PortalKeyPath != "" {
+		if err := in.installPortalManager(in.stosConfig.Spec.Install.PortalKeyPath, fsStosClusterName, in.stosConfig.Spec.Install.StorageOSClusterNamespace); err != nil {
+			return err
+		}
+	}
 
 	err = in.kustomizeAndApply(filepath.Join(stosDir, clusterDir), stosClusterFile)
 
 	return err
 }
 
+func (in *Installer) installPortalManager(keyPath, stosClusterName, stosClusterNS string) error {
```

Comment:
I thunk this function breaks single responsibility, because it not only installs portal manager, but updates cluster config too. Maybe a better name also should solve this problem.

Reply:
Yes, I went back and forth on this a little. I think I will just move this functionality back up to `installStorageOS` - it makes more sense.